### ISSUE TITLE
Remove a container not working with docker 1.13.0

### ISSUE THREAD

**Issue by GabyCT** [01org/cc-oci-runtime#548]
There are some issues in removing a container using docker 13.0. Tests were conducted using Fedora 24  for CC 2.1 on a baremetal.

When you are doing a docker rm -f $(docker ps -qa), we are obtaining:

- check that replicas has two interfaces
   (from function `clean_docker_ps' in file ../../lib//test-common.bash, line 29,
    from function `setup' in test file swarm.bats, line 43)
     `clean_docker_ps' failed with status 123
   67228d33a62e
   d5c16db83e2d
   5568e6c74270
   Error response from daemon: Unable to remove filesystem for 67228d33a62e6116066e434f5fe109446d0567569440fd39f17f0226dd100b71: remove /var/lib/docker/containers/67228d33a62e6116066e434f5fe109446d0567569440fd39f17f0226dd100b71/shm: device or resource busy
   Error response from daemon: No such container: d5c16db83e2d
   Error response from daemon: Unable to remove filesystem for 5568e6c74270c252a65746304ff7f3786bfbe71c530205455f102e13fa3b1374: remove /var/lib/docker/containers/5568e6c74270c252a65746304ff7f3786bfbe71c530205455f102e13fa3b1374/shm: device or resource busy

**Comment by gorozco1** [comment]
uhmm did it work with docker-engine-1.12.1-0.2.rc2.fc24.x86_64 ?

**Comment by GabyCT** [comment]
@gorozco1 yes it work with docker 1.12.1
bats swarm.bats
 ✓ check that the replicas' names are different
 ✓ check that replicas has two interfaces
 ✓ check service ip among the replicas
 ✓ ping among replicas on their gateway ip
 ✓ quick ping among replicas on their overlay ip
but not with newer docker versions

**Comment by GabyCT** [comment]
This behavior is currently showed in docker 1.13

**Comment by devimc** [comment]
seems like this bug was fixed in 1d32b9ce607130480cbe76d8f6cf503f3b00e2c

```
Docker version 1.13.0-rc7, build 48a9e53
containerd version 0.2.4 commit: 71861281661331fbc81936be81f05d8db71fc1ca
cc-oci-runtime commit: 37523e110c7c6abf15af223b146bcf854102935b
```
